src/components/common/RadioGroup.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <p>{{ groupLabel }}</p>
+  <div class="flex justify-between items-center">
+    <div v-for="option in options" :key="option.key">
+      <input
+        class="mr-1"
+        type="radio"
+        :name="option.name"
+        :id="option.key"
```

Comment:
potrzebujesz do czegoś konkretnego tego id?

Reply:
Tylko po to, żeby po kliknięciu na label zaznaczał się radio button. Zamiast `option.key` skorzystałem z indeksowania w `v-for`.
```vue
<div v-for="(option, index) in options" :key="option.key">
  <input
    :name="option.name"
    :id="index"
    :value="option.value"
    class="mr-1"
    type="radio"
    v-model="selected"
  />
  <label :for="index">{{ option.label }}</label>
</div>
```